tests/unit/cli/test_types.py

```diff
@@ -241,6 +241,64 @@ def test_successful_parsing(self, input, expected):
         self.assertEqual(result, expected, msg="Failed with Input = " + str(input))
 
 
+class TestCfnTagsMultipleValues(TestCase):
+    """
+    Tests for the CfnTags parameter allowing multiple values per key.
+    """
+
+    def setUp(self):
+        self.param_type = CfnTags(multiple_values_per_key=True)
+
+    @parameterized.expand(
+        [
+            # Just a string
+            ("some string"),
+            # Wrong notation
+            # ("a==b"),
+            # Wrong multi-key notation
+            # ("a==b,c==d"),
```

Comment:
Should line 257 and 259 be commented?

Reply:
Those are commented in many other tests in that file, I've only included them for consistency. Maybe they're going to be re-enabled in the future? If we remove these I think we should remove them all.